core/src/components/side-menu/side-menu-user-image/side-menu-user-image.tsx

```diff
@@ -5,9 +5,8 @@ import { Component, h, Host, Prop } from '@stencil/core';
 // FIXME: Can the logic for it be directly integrated in side-menu-user instead?
 
 /**
- * @slot <default> -
- * <b>Unnamed slot.</b> Used as alternative to props to inject <code><img...</code> element directly into the DOM.
- * */
+ * @slot <default> - <b>Unnamed slot.</b> Used as alternative to props to inject <code><img...</code> element directly into the DOM.
```

Comment:
Side Menu User Image is not present as a tab under the Notes tab in Storybook, should it be added?

Reply:
I don't think they need to be. They are (internal) components of the <tds-side-menu-user>. I think its more bad than good if we document them tbh, since then we cant change them.
Perhaps they should be renamed with `core-` or something in their names? Ill add a check in topic.

Reply:
To on what Alex wrote, as those are internal components for other component, I think it is OK not to display them. For those are curiouse, there is always part od readme which states what is component dependent on: https://github.com/scania-digital-design-system/tegel/tree/develop/packages/core/src/components/side-menu/side-menu-user

It is good idea to have different prefix so we know it is internaly used component.